src/Symfony/Component/Security/Http/composer.json

```diff
@@ -18,12 +18,13 @@
     "require": {
         "php": ">=7.2.5",
         "symfony/deprecation-contracts": "^2.1|^3",
-        "symfony/security-core": "^5.4.19|~6.0.19|~6.1.11|^6.2.5",
         "symfony/http-foundation": "^5.3|^6.0",
         "symfony/http-kernel": "^5.3|^6.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php80": "^1.16",
-        "symfony/property-access": "^4.4|^5.0|^6.0"
+        "symfony/property-access": "^4.4|^5.0|^6.0",
+        "symfony/service-contracts": "^1.10|^2|^3",
+        "symfony/security-core": "^5.4.19|~6.0.19|~6.1.11|^6.2.5"
```

Comment:
This line contains no diff, except the trailing `,`. Can you please move it back up? Thanks